content/applications/inventory_and_mrp/inventory/product_management/product_tracking/create_sn.rst

```diff
@@ -0,0 +1,193 @@
+   **only** on a receipt.
+
+   .. figure:: create_sn/purchase-order-or-receipt.png
+      :align: center
+      :alt: Show the purchase order and the Receipt smart button
+
+      Screenshot of a "Purchase Order", with the "Receipt" smart button at the top.
+
+   To record an item's serial number before receiving the item, follow the steps in the next
+   sections to assign serial numbers, but do **not** click the receipt's :guilabel:`Validate` button
+   until the products are received from the vendor.
+
+Assign a single serial number to a product by clicking the :guilabel:`Detailed Operations` smart
+button on a receipt.
+
+In the :guilabel:`Lot/Serial Number Name` field, manually type in the serial number for a single
+product.
+
+.. image:: create_sn/add-a-line.png
+   :align: center
+   :alt: Add a line on the Detailed Operations page to assign serial numbers.
+
+When finished, click the receipt's breadcrumbs, and the assigned serial numbers are automatically
+saved.
+
+.. _inventory/product_management/assign-sn:
+
+Assign serial numbers
+=====================
+
+To generate new serial numbers in a sequence, click the :guilabel:`+ (plus)` icon in the
+:ref:`product line <inventory/product_management/detailed-operations-popup>`.
+
+.. important::
+   If the icon is not visible, ensure the :guilabel:`Create New` option is selected in the
+   :ref:`receipt's configuration page <inventory/product_management/configure-new-serials>`.
+
+Doing so opens the :guilabel:`Assign Serial Numbers` pop-up window. The :guilabel:`Number of SN`
+field is auto-filled based on the product quantity requiring serial numbers. Manually input the
+first serial number in the :guilabel:`First SN` field, and click :guilabel:`Assign Serial Numbers`
+to generate a sequence of serial numbers based on the first serial number entered.
+
+.. image:: create_sn/assign-numbers-in-sequence.png
+   :align: center
+   :alt: Show Assign Serial numbers pop-up.
+
+.. _inventory/product_management/stock-move-section:
+
+Stock move pop-up
+=================
+
+For various methods of assigning serial numbers in bulk, click the :guilabel:`⦙≣ (bulleted list)`
+icon in the :ref:`product line <inventory/product_management/detailed-operations-popup>` of a
+receipt.
+
+Add a line
+----------
+
+In the :guilabel:`Open: Stock move` pop-up, manually input the serial numbers in the
+:guilabel:`Lot/Serial Number` column.
+
+.. image:: create_sn/add-a-line-stock-move.png
+   :align: center
+   :alt: Add a line on the Stock move pop-up.
+
+Generate Serials
+----------------
+
+Assign multiple serial numbers at once by clicking the :guilabel:`Generate Serials/Lots` button in
+the :guilabel:`Open: Stock move` pop-up window.
+
+.. image:: create_sn/generate-serials.png
+   :align: center
+   :alt: Show generate serials button.
+
+Doing so opens the :guilabel:`Assign Serial Numbers` pop-up window, where the first serial number is
```

Comment:
```suggestion
Doing so opens the :guilabel:`Generate Serial numbers` pop-up window, where the first serial number is
```
this is how the pop-up window is titled on the 17.0 runbot I'm using -- may be worth double-checking during your revisions just to be sure

Reply:
Thanks for pointing this out! I must have gotten this window confused with the other serial numbers window in the other section! Appreciate your help